base/deprecated.jl

```diff
@@ -268,5 +268,6 @@ end
 
 @deprecate var"@_inline_meta"   var"@inline"   false
 @deprecate var"@_noinline_meta" var"@noinline" false
+@deprecate getindex(@nospecialize(t::Tuple), i::Real) = t[convert(Int, i)]
```

Comment:
> anything else to do here

Make sure Julia builds and passes tests? 🙂 

```suggestion
@deprecate getindex(@nospecialize(t::Tuple), i::Real) t[convert(Int, i)]
```

Reply:
that might be a good idea.